docs/en/contribution/How-to-release.md

```diff
@@ -183,11 +189,27 @@ Vote result should follow these:
     svn mv https://dist.apache.org/repos/dist/dev/skywalking/python/"$VERSION" https://dist.apache.org/repos/dist/release/skywalking/python/"$VERSION"
     ```
     
-1. Refer to the previous [PR](https://github.com/apache/skywalking-website/pull/132), update news and links on the website. There are several files need to modify.
+2. Refer to the previous [PR](https://github.com/apache/skywalking-website/pull/571), update news and links on the website. There are several files need to modify.
+
+
+### Publish PyPI package
+
+After the official ASF release, we publish the packaged wheel to the PyPI index.
+
+1. Make sure the final upload is correct by using the test PyPI index `make upload-test`.
+2. Upload the final artifacts by running `make upload`.
+
+### Publish Docker images
+
+After the release of PyPI packages, we build Docker images based on the wheels.
```

Comment:
The Docker images are not based on the PyPI wheels now, the Dockerfile builds the wheels on the fly and install from the local wheels to build the Docker images, we did this since we build Docker images for multiple architectures

Reply:
I see, lemme change this and remove the other docker doc then.